db/migrate/20161109150714_add_templates_to_resolve_url.rb

```diff
@@ -0,0 +1,21 @@
+class AddTemplatesToResolveUrl < ActiveRecord::Migration[5.0]
```

Comment:
Yea, for migrations that are really just transformations and don't affect DB schema, this seems like a good way to test them.